bodyshopbidsdotcom/git-scripts
# w278177436 Remove default master branch parameter from git-scripts

Closes https://www.wrike.com/open.htm?id=278177436
Fixes 
https://github.com/bodyshopbidsdotcom/git-scripts/issues/396
https://github.com/bodyshopbidsdotcom/git-scripts/issues/397
https://github.com/bodyshopbidsdotcom/git-scripts/issues/404

## What
Pull the default branch from GitHub API instead of `.git-scripts.yml` file 
## Why
Teams don't have to update their `.git-scripts.yml` file if they change default branch 
## Testing

### How to reproduce error?
1. Checkout `release` branch of `git-scripts`
2. Run workon git-scripts and run pip install -r requirements.txt
3. Checkout `vice-frontend`, run `git_create_pr`. Note: `vice-frontend` default branch is release and it doesn't have any `.git-scripts.yml` file
4. You should see `GitCommandError` 

### Now how to test the fix?
1. Checkout `this` branch of `git-scripts`
2. Run `workon git-scripts` and run `pip install -r requirements.txt`
3. Checkout `vice-frontend`, run `git_create_pr`
4. You should not see any error 

Run below commands for regression:
1. `git_update_current`
2. `git_check_for_updates`
3. `git_create_pr`
